components/automate-ui/src/app/modules/infra-proxy/create-environment-modal/create-environment-modal.component.html

```diff
@@ -0,0 +1,109 @@
+<chef-modal [visible]="visible" (closeModal)="closeCreateModal()">
+  <div class="text-right">
+    <h2 slot="title">Create Environment</h2>
+  </div>
+  <div class="flex-container">
+    <div class="main pages-body">
+      <app-infra-tab-change class="tabs-row" (tabChange)="tabChange($event)">
+        <app-infra-tab tabTitle="Details" [active]="detailsTab" [disabled]="false">
+          <form [formGroup]="detailsFormGroup">
+            <div class="input-margin">
+              <chef-form-field>
+                <label>
+                  <span class="label">Name <span aria-hidden="true">*</span></span>
+                  <input chefInput formControlName="name" type="text" id="name-input" (keyup)="handleInput($event)" autocomplete="off"/>
+                </label>
+                <chef-error
+                  *ngIf="(detailsFormGroup.get('name').hasError('required') || detailsFormGroup.get('name').hasError('pattern')) && detailsFormGroup.get('name').dirty">
+                  Display Name is required.
```

Comment:
Must match the text in L13:
```suggestion
                  Name is required.
```

Reply:
updated changes